### ISSUE TITLE
ONNX - Operators

### ISSUE THREAD

**Issue by junxnone** [junxnone/tio#738]
# Reference
- [ ] [Neural Network Operators](https://github.com/onnx/onnx/blob/master/docs/Operators.md)
- [ ] [Classical Machine Learning operators](https://github.com/onnx/onnx/blob/master/docs/Operators-ml.md)
- [ ] [defines code](https://github.com/onnx/onnx/tree/master/onnx/defs)

# Brief
- Neural Network Operators
- Classical Machine Learning operators

## Classical Machine Learning operators
- ai.onnx.ml

Name | Description | Input| Output
-- | -- | -- | --
ai.onnx.ml.ArrayFeatureExtractor
ai.onnx.ml.Binarizer
ai.onnx.ml.CastMap
ai.onnx.ml.CategoryMapper
ai.onnx.ml.DictVectorizer
ai.onnx.ml.FeatureVectorizer
ai.onnx.ml.Imputer
ai.onnx.ml.LabelEncoder
ai.onnx.ml.LinearClassifier
ai.onnx.ml.LinearRegressor
ai.onnx.ml.Normalizer
ai.onnx.ml.OneHotEncoder
ai.onnx.ml.SVMClassifier
ai.onnx.ml.SVMRegressor
ai.onnx.ml.Scaler
ai.onnx.ml.TreeEnsembleClassifier
ai.onnx.ml.TreeEnsembleRegressor
ai.onnx.ml.ZipMap

## Neural Network Operators
- ai.onnx 

Name | Description | Input | Output
-- | -- | -- | --
Abs | 绝对值 | Tensor | Tensor
Acos | 反余弦 | Tensor | Tensor
Acosh
Add
And
ArgMax
ArgMin
Asin
Asinh
Atan
Atanh
AveragePool
BatchNormalization
BitShift
Cast
Ceil
Clip
Compress
Concat
ConcatFromSequence
Constant
ConstantOfShape
Conv
ConvInteger
ConvTranspose
Cos
Cosh
CumSum
DepthToSpace
DequantizeLinear
Det
Div
Dropout
Einsum
Elu
Equal
Erf
Exp
Expand
EyeLike
Flatten
Floor
GRU
Gather
GatherElements
GatherND
Gemm
GlobalAveragePool
GlobalLpPool
GlobalMaxPool
Greater
HardSigmoid
Hardmax
Identity
If
InstanceNormalization
Inverse
IsInf
IsNaN
LRN
LSTM
LeakyRelu
Less
Log
LogSoftmax
Loop
LpNormalization
LpPool
MatMul
MatMulInteger
Max
MaxPool
MaxRoiPool
MaxUnpool
Mean
Min
Mod
Mul
Multinomial
Neg
NonMaxSuppression
NonZero
Not
OneHot
Or
PRelu
Pad
Pow
QLinearConv
QLinearMatMul
QuantizeLinear
RNN
RandomNormal
RandomNormalLike
RandomUniform
RandomUniformLike
Reciprocal
ReduceL1
ReduceL2
ReduceLogSum
ReduceLogSumExp
ReduceMax
ReduceMean
ReduceMin
ReduceProd
ReduceSum
ReduceSumSquare
Relu
Reshape
Resize
ReverseSequence
RoiAlign
Round
Scan
Scatter
ScatterElements
ScatterND
Selu
SequenceAt
SequenceConstruct
SequenceEmpty
SequenceErase
SequenceInsert
SequenceLength
Shape
Shrink
Sigmoid
Sign
Sin
Sinh
Size
Slice
Softmax
Softplus
Softsign
SpaceToDepth
Split
SplitToSequence
Sqrt
Squeeze
StringNormalizer
Sub
Sum
Tan
Tanh
TfIdfVectorizer
ThresholdedRelu
Tile
TopK
Transpose
UnfoldToDepth
Unique
Unsqueeze
Upsample
Where
Xor

**Comment by junxnone** [comment]
#734
